ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.html

```diff
@@ -0,0 +1,155 @@
+<div class="Wrapper">
```

Comment:
Nothing was changed in the file.
Just extracted the template into separate file.